tests/MongoDB.Driver.Tests/Specifications/server-discovery-and-monitoring/tests/integration/connectTimeoutMS.yml

```diff
@@ -0,0 +1,88 @@
+# Test SDAM error handling.
```

Comment:
Refresh this file from the spec repo as it has changed since being pulled in. Notably `hello` was added to the list of fail commands.